__tests__/tx-template.test.js

```diff
@@ -1,5 +1,6 @@
 import { HathorWallet, SendTransaction } from '@hathor/wallet-lib';
 import TestUtils from './test-utils';
+// eslint-disable-next-line import/named
```

Comment:
thought: Weird that we have to use this rule cancellation. Looking at the [rule docs](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/named.md) it seems we are just writing the constants in a way that's unreachable for eslint.

Maybe we should rewrite this constants file in the future so that no runtime changes happen to the constants, and eslint is able to identify all properties statically.

Reply:
This is an issue on some other files and the LSP as well, it does not seem to find the exported constants.
I think its because this is one of the files that still use `module.exports` syntax.